modules/upgrading-mtc-1-8-0.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/upgrading-mtc.adoc
+
+:_content-type: PROCEDURE
+[id="migration-upgrading-mtc-18_{context}"]
+= Upgrading to {mtc-full} to 1.8.0
+
+
+.Procedure
+
+* Determine subscription names and current channels to work with for upgrading, by either issuing:
+
++
+[source,terminal]
+----
+$ oc -n openshift-migration get sub
+----
++
+Which will return an output similar to the following:
+
++
+[source,terminal]
+----
+NAME                                                                         PACKAGE                SOURCE                 CHANNEL
+mtc-operator                                                                 mtc-operator           mtc-operator-catalog   release-v1.7
+redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace   redhat-oadp-operator   mtc-operator-catalog   stable-1.0
+----
++
+Or, for a more programmatic approach, issuing:
++
+[source,terminal]
+----
+$ oc -n openshift-migration get sub -ojson | jq -r '.items[] | { name: .metadata.name, package: .spec.name, channel: .spec.channel }'
+----
++
+Which will return an output similar to the following:
++
+[source,terminal]
+----
+{
+  "name": "mtc-operator",
+  "package": "mtc-operator",
+  "channel": "release-v1.7"
+}
+{
+  "name": "redhat-oadp-operator-stable-1.0-mtc-operator-catalog-openshift-marketplace",
+  "package": "redhat-oadp-operator",
+  "channel": "stable-1.0"
+}
+----
+
+* For each subscription, patch to move from the {mtc-short} 1.7 channel to the newer channel {mtc-short} 1.8:
++
+[source,terminal]
+----
+$ oc -n openshift-migration patch subscription mtc-operator --type merge --patch '{"spec": {"channel": "release-v1.8"}}'
+----
++
+The subscription is returned as `patched`:
++
+[source,terminal]
+----
+subscription.operators.coreos.com/mtc-operator patched
+----
```

Comment:
This block is out-dented from the rest of the bullet content preceding it when looking at the preview.

Reply:
I think it is ok in the preview:

![image](https://github.com/openshift/openshift-docs/assets/106804821/2438444e-c213-42d5-ae5e-83103cea8919)

Reply:
This too now shows up as expected for me when I preview today.